perfmetrics/scripts/testing_on_gke/examples/fio/parse_logs.py

```diff
@@ -231,45 +211,47 @@ def downloadFioOutputs(fioWorkloads: set, instanceId: str):
           per_epoch_output_data["jobs"][0]["job_start"]
       )
       r["end"] = unix_to_timestamp(per_epoch_output_data["timestamp_ms"])
-      if r["scenario"] != "local-ssd" and mash_installed:
-        r["lowest_memory"], r["highest_memory"] = get_memory(
-            r["pod_name"],
-            r["start"],
-            r["end"],
-            project_number=args.project_number,
-        )
-        r["lowest_cpu"], r["highest_cpu"] = get_cpu(
-            r["pod_name"],
-            r["start"],
-            r["end"],
-            project_number=args.project_number,
-        )
-        pass
+
+      if r["scenario"] != "local-ssd":
+        if mash_installed:
+          r["lowest_memory"], r["highest_memory"] = get_memory(
+              r["pod_name"],
+              r["start"],
+              r["end"],
+              project_number=args.project_number,
+          )
+          r["lowest_cpu"], r["highest_cpu"] = get_cpu(
+              r["pod_name"],
+              r["start"],
+              r["end"],
+              project_number=args.project_number,
+          )
+
       r["gcsfuse_mount_options"] = gcsfuse_mount_options
       r["blockSize"] = bs
       r["filesPerThread"] = nrfiles
       r["numThreads"] = numjobs
 
+      # This print is for debugging in case something goes wrong.
       pprint.pprint(r)
 
+      # If a slot for record for this particular epoch has not been created yet,
+      # append enough empty records to make a slot.
       while len(output[key]["records"][scenario]) < epoch:
         output[key]["records"][scenario].append({})
 
+      # Insert the record at the appropriate slot.
       output[key]["records"][scenario][epoch - 1] = r
 
-  scenario_order = [
-      "local-ssd",
-      "gcsfuse-generic",
-      "gcsfuse-no-file-cache",
-      "gcsfuse-file-cache",
-  ]
+  return output
 
-  output_file_path = args.output_file
-  ensureDir(os.path.dirname(output_file_path))
-  output_file = open(output_file_path, "a")
-  output_file.write(
+
+def writeRecordsToCsvOutputFile(output: dict, output_file_path: str):
+  output_file_fwr = open(output_file_path, "a")
```

Comment:
Use "with" statement to manage resources better.
```suggestion
  with open(output_file_path, "a") as output_file_fwr:
```

Reply:
done